R/compile-dll.R

```diff
@@ -10,7 +10,9 @@
 #' @note If this is used to compile code that uses Rcpp, you will need to
 #'   add the following line to your `Makevars` file so that it
 #'   knows where to find the Rcpp headers:
-#'   `PKG_CPPFLAGS=`$(R_HOME)/bin/Rscript -e 'Rcpp:::CxxFlags()'``
```

Comment:
Currently, this is incorrectly formatted: https://r-lib.github.io/pkgbuild/reference/compile_dll.html#note

Also, I am not sure what the `{cpp11}` equivalent is here, or if it should even be mentioned.